knowledge_repo/app/routes/web_editor.py

```diff
@@ -35,43 +35,21 @@
 
 @blueprint.route('/ajax_tags_typeahead', methods=['GET'])
 def generate_tags_typeahead():
-    tag_objs = (db_session.query(Tag).all())
-    tags_typeahead = [str(tag_obj.name) for tag_obj in tag_objs]
-    return json.dumps(tags_typeahead)
+    return json.dumps([t[0] for t in db_session.query(Tag.name).all()])
```

Comment:
Yeah I don't like the t[0] either, but this is in fact more efficient, given that the query only pulls the name, so I think we should keep it as is?